config/samples/go_example_loadtest.yaml

```diff
@@ -50,14 +50,67 @@ spec:
   # results:
   #   bigQueryTable: "example-project.foo.demo_dataset"
 
-  # Scenarios are separate objects under this proposal. While
-  # they specify configuration for clients and servers, they are
-  # highly specialized. I doubt we want to modify controller logic
-  # whenever the Scenario proto is changed. This is open to
-  # discussion, however.
-  #
-  # These can be stored in ConfigMaps or they can be high-level
-  # Kubernetes objects themselves. If we did want to couple, they
-  # could also be inlined.
-  scenarios:
-    - name: go-example-scenario
+  # ScenariosJSON is string with the contents of a Scenarios message, formatted
+  # as JSON. See the Scenarios protobuf definition for details:
+  # https://github.com/grpc/grpc-proto/blob/master/grpc/testing/control.proto.
+  scenariosJSON: |
+    {
+      "scenarios": [
+        {
+          "name": "go_example_scenario",
```

Comment:
nit: the C++ embeded example has a name of : ` cpp_generic_async_streaming_ping_pong_secure` - perhaps we want to use the name of the original scenario_config.py scenario here as well?